lessons/220/ruby-app/config.ru

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'connection_pool'
+require 'iodine'
+require 'oj'
+require 'pg'
+require 'prometheus/middleware/exporter'
+require 'rack'
+require 'yaml'
+
+require_relative 'src/app'
+
+RubyVM::YJIT.enable
+Iodine.threads = ENV.fetch('RAILS_MAX_THREADS', 2).to_i
+Iodine.workers = ENV.fetch('WORKERS_NUM', 2).to_i
+Oj.default_options = { mode: :compat }
+pg_conf = YAML.load(File.read('db/config.yml'), aliases: true)['production']
+PG_POOL = ConnectionPool.new(size: Iodine.threads * Iodine.workers, timeout: 5) do
+  PG.connect(pg_conf['host'], pg_conf['port'], nil, nil, pg_conf['database'], pg_conf['username'], pg_conf['password'])
```

Comment:
@simpl1g good pr, we can also optimize the work with database through prepared queries via mini-sql gem

https://github.com/discourse/mini_sql?tab=readme-ov-file#prepared-statements

```ruby
SQL = <<~SQL
  INSERT INTO "devices" ("uuid", "mac", "firmware", "created_at", "updated_at")
  VALUES (?, ?, ?, ?, ?)
  RETURNING "id"
SQL

PG_POOL.with do |mini_sql|
  device.id = mini_sql.prepared.query_single(SQL, device.uuid, device.mac, device.firmware, device.created_at, device.updated_at).first
end
```
prepared queries give a big win when there are a lot of joins, here I think we can get a 10% boost speed

Reply:
@ermolaev thanks! I also thought about it, but node app doesn't use prepared statements, so I decided not to use them as well

Reply:
ok thanks, i used in some apps but not in node